### ISSUE TITLE
Package name

### ISSUE THREAD

**Issue by mazzma12** [taynaud/python-louvain#23]
Why is the package listed as `python-louvain` but you have to use it with `import community` ? 

* It is misleading as people try to do `pip install community` 
* It shadows the name of the other `community` python package .

You run into troubles if you use both at the same time. Which is likely to occur if people ran `pip install community` in the first place

**Comment by taynaud** [comment]
Hi,

This is only for historical reason (I start this way, not really thinking about a distribution on pypi).

I know this issue, but it is a bit complicated to change the name. For such a small package, breaking the API can be complicated. 

Shadowing the community package is a bigger concern than installing python-louvain to get the community package. Do you have better idea than publishing another pypi package with the same code in another package name ?

If you do not use both 'community' package in the same software, virtual environments may be a solution ?

Best,

**Comment by mazzma12** [comment]
> Do you have better idea than publishing another pypi package with the same code in another package name ?

Yeah, you can stick to your name `python-louvain` on the PyPi repo. You shall just add a first folder 'python_louvain' at the root that would contain the 'community' folder.

This way you just have to refactor your imports into : `from python_louvain import community`

> If you do not use both 'community' package in the same software, virtual environments may be a solution ?

I totally agree about virtualenv. The problem is that people may find your package in another snippet of code and see `import community` and then run `pip install community` which will shadow the name unless they uninstall it.

**Comment by pavlin-policar** [comment]
To avoid backwards compatibility issues, you may want to move all the code to another module e.g. `python_louvain`, but also leave the community file intact for the time being, so as not to break anything that currently depends on it. So in `community.py` you can simply import the relevant functions from `python_louvain.py`. And when somebody imported `community` you may want to trigger a `DeprecationWarning`.

Then remove `community` altogether. This will give people depending on this library a heads up.

As a side note, are you planning a release anytime soon?

**Comment by taynaud** [comment]
I have just make a release, I will consider renaming or having both name in next version

**Comment by thisisreallife** [comment]
Problem is still unsolved in Mar, 2021

**Comment by thisisreallife** [comment]
one solution is uninstall 'community' and then install 'python_louvain' and 'networkx': https://stackoverflow.com/questions/53087066/how-to-use-the-communities-module-python-louvain-in-networkx-2-2/53101996
